fast/stages/0-bootstrap/data/org-policies/compute.yaml

```diff
@@ -28,6 +28,29 @@ compute.vmExternalIpAccess:
   rules:
     - deny:
         all: true
+
+# only allow GCP images by default
+compute.trustedImageProjects:
+  rules:
+  - allow:
+      values:
+      - "projects/centos-cloud"
+      - "projects/cos-cloud"
+      - "projects/debian-cloud"
+      - "projects/fedora-cloud"
+      - "projects/fedora-coreos-cloud"
```

Comment:
Those two are community supported. If we want to include all community supported then we should add:
* almalinux-cloud
* freebsd-org-cloud-dev 
* opensuse-cloud
* cloud-hpc-image-public

Source: https://cloud.google.com/compute/docs/images#os-compute-support

And for previous cooment: https://cloud.google.com/compute/docs/images/os-details

Reply:
Send a PR after merge to include all community supported, but leaving them commented.